encoding/serialization.go

```diff
@@ -35,6 +35,9 @@ func (c *Frame) SerializeGnark() ([]byte, error) {
 }
 
 func (c *Frame) DeserializeGnark(data []byte) (*Frame, error) {
+	if len(data) < bn254.SizeOfG1AffineCompressed {
```

Comment:
it can be tightened as `len(data) <= bn254.SizeOfG1AffineCompressed`, as the coeffs cannot be empty